lib/core/src/server/config/style.config.js

```diff
@@ -0,0 +1,42 @@
+import { logger } from '@storybook/node-logger';
+import autoprefixer from 'autoprefixer';
+import findUp from 'find-up';
+
+export default function getStyleConfig(baseConfig) {
+  // null if file is not found
+  const customPostcssConfig = findUp.sync('postcss.config.js', {
```

Comment:
Will be very nice if you could asyncify this... Need to add some additional `await`s in the call chain though. Not critical one, bit since we already support async/await that be great =)